hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OmSnapshotManager.java

```diff
@@ -67,21 +77,74 @@ public final class OmSnapshotManager implements AutoCloseable {
   private final OzoneManager ozoneManager;
   private final SnapshotDiffManager snapshotDiffManager;
   private final LoadingCache<String, OmSnapshot> snapshotCache;
-  private final ManagedRocksDB snapshotDiffDb;
+  private ManagedRocksDB snapshotDiffDb;
+
+  /**
+   * Contains all the snap diff job which are either queued, in_progress or
+   * done. This table is used to make sure that there is only single job for
```

Comment:
When do we remove the job entries from the table  ?

Reply:
It would be time based (keep the diff report for 7 days) or status based (Failed or Rejected).